pkg/client/client_test.go

```diff
@@ -2199,1464 +2205,1765 @@ U5wwSivyi7vmegHKmblOzNVKA5qPO8zWzqBC
 		})
 	})
 
-	Describe("List", func() {
-		Context("with structured objects", func() {
-			It("should fetch collection of objects", func() {
-				By("creating an initial object")
-				dep, err := clientset.AppsV1().Deployments(ns).Create(ctx, dep, metav1.CreateOptions{})
-				Expect(err).NotTo(HaveOccurred())
-
-				cl, err := client.New(cfg, client.Options{})
-				Expect(err).NotTo(HaveOccurred())
-
-				By("listing all objects of that type in the cluster")
-				deps := &appsv1.DeploymentList{}
-				Expect(cl.List(context.Background(), deps)).NotTo(HaveOccurred())
+	Describe("CreateOptions", func() {
+		It("should allow setting DryRun to 'all'", func() {
+			co := &client.CreateOptions{}
+			client.DryRunAll.ApplyToCreate(co)
+			all := []string{metav1.DryRunAll}
+			Expect(co.AsCreateOptions().DryRun).To(Equal(all))
+		})
 
-				Expect(deps.Items).NotTo(BeEmpty())
-				hasDep := false
-				for _, item := range deps.Items {
-					if item.Name == dep.Name && item.Namespace == dep.Namespace {
-						hasDep = true
-						break
-					}
-				}
-				Expect(hasDep).To(BeTrue())
-			})
+		It("should allow setting the field manager", func() {
+			po := &client.CreateOptions{}
+			client.FieldOwner("some-owner").ApplyToCreate(po)
+			Expect(po.AsCreateOptions().FieldManager).To(Equal("some-owner"))
+		})
 
-			It("should fetch unstructured collection of objects", func() {
```

Comment:
This test was redundant with the one in the unstructured section, so I deleted it